File: languages/C++/src/Girl.cpp
#include "Girl.hpp"
#include "Person.hpp"
#include <iostream>

void Girl::kiss(Person someone)
{
   std::cout<<"「"<<this->getName()<<"」 亲了下 「"<<someone.getName()<<"」。"<<std::endl;
}

void Girl::fallInLoveWith(Person someone)
{
   std::cout<<"「"<<this->getName()<<"」 与 「"<<someone.getName()<<"」 坠入爱河。"<<std::endl;;
}

File: languages/JavaScript/lib/Girl.js
import Person from "./Person";

/**
 * 女孩子
 */
export default class Girl extends Person {
  /**
   * Kiss
   */
  kiss(someone) {
    console.log(`「${this.name}」 亲了下 「${someone.name}」。`);
  }

  /**
   * 与某人坠入爱河
   * @param someone 某人
   */
  fallInLoveWith(someone) {
    console.log(`「${this.name}」 与 「${someone.name}」 坠入爱河。`);
  }
}
